server/spec/autobind_disabled_for_owner_spec.rb

```diff
@@ -154,4 +154,26 @@
     job['state'].should == "FAILED"
     job['resultData'].should == "org.candlepin.async.JobExecutionException: Auto-attach is disabled for owner #{owner['key']} because of the content access mode setting."
   end
+
+  it "should not dryrun when owner is in SCA mode" do
```

Comment:
The premise of this test is also wrong, and it actually exposes a gap we missed.
When we did the changes to no-op  auto-attach and return status 200 when in SCA mode (https://github.com/candlepin/candlepin/pull/2947), we should have _also_ applied that change to the dry-run auto-attach. Let's apply that change now, and change this test to assert the opposite: that dry-run-auto-attach returns 200 when on SCA mode

Reply:
Updated.